Tests/XCUITests/IntegrationTests.swift

```diff
@@ -48,14 +48,15 @@ class IntegrationTests: BaseTestCase {
             return true
         }
         app.swipeDown()
+        sleep(5)
```

Comment:
Unrelated to this PR's subject: Clicking "Allow" from the notification before proceeding to the following step seem to stablize the tessts.

Reply:
The lack of wait may be exposing an issue on Firefox itself.